.gitignore

```diff
@@ -8,4 +8,5 @@
 **/.vscode
 
 # node
-**/node_modules
\ No newline at end of file
+**/node_modules
+package-lock.json
```

Comment:
Why did you add this file to `.gitignore` and simultaneously commited it to the repo?